neo.UnitTests/SmartContract/UT_JsonSerializer.cs

```diff
@@ -119,7 +119,7 @@ public void JsonTest_String()
             var json = @" ["""" ,  ""\b\f\t\n\r\/\\"" ]";
             var parsed = JObject.Parse(json);
 
-            Assert.AreEqual(@"["""",""\b\f\t\n\r\/\\""]", parsed.ToString());
+            Assert.AreEqual(@"["""",""\b\f\t\n\r/\\""]", parsed.ToString());
```

Comment:
Hi @shargon , I'm curious: what is the difference here and in the exception type?

Reply:
Checking... because i don't know why these changes right now...

Reply:
It's because `\/` in c# was escaped as '/' and then you need to check without the slash, but i don't know why the change here. ...

Reply:
I see. Also: why are we changing the exception type?

Reply:
Is not changed, in `double.Parse` if you send a big number, it throw an `OverflowException` maybe it was changed in dotnet 3?

Reply:
I have no idea haha, we can keep this change if you prefer, I'm just surprised that the test results didn't change.